lib/v2/mangadex/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'mangadex.org': {
+        _name: 'MangaDex',
+        '.': [
+            {
+                title: '漫画更新',
+                docs: 'https://docs.rsshub.app/anime.html#mangadex',
+                source: '/title/:id',
```

Comment:
```suggestion
                source: ['/title/:id/*', '/title/:id'],
```
This will match `https://mangadex.org/title/58be6aa6-06cb-4ca5-bd20-f1392ce451fb/yotsuba` as well